templates/upload.html

```diff
@@ -9,13 +9,14 @@ <h4 class="modal-title">Upload Data</h4>
       <col width="100">
     </colgroup>
     <tr>
-      <td>Data Type</td>
+      <td style="white-space:nowrap">Data Type</td>
```

Comment:
Please try to set this in css later. Inline css is to be avoided unless for initial property such as display none.